autogen.sh

```diff
@@ -1,7 +1,6 @@
 #!/bin/sh
 set -e
 
-./update.sh
```

Comment:
keep the update.sh, i'm actually using it!
It still needs some fixing, i'll get to it.
For now do
  mkdir openbsd
 cd openbsd
 ln -s /usr/src ./
before you run ./autogen.sh